src/views/Dashboard.vue

```diff
@@ -155,17 +155,17 @@ export default {
   computed: {
     userId: function () { return this.$store.getters.userId },
     communityId: function () { return this.$store.getters.communityId },
-    membersNotInvited: function() {
-      let list = []
+    membersNotInvited: function () {
+      const list = []
```

Comment:
Please test if this works as constant, because we are changing the list few rows below. And as a whole you can't create constant for array that you intend to modify immediately.